packages/@react-spectrum/dnd/docs/dnd.mdx

```diff
@@ -292,23 +292,18 @@ function DroppableList(props) {
 ## API
 
 As seen in the [example](#example) above, enabling drag and drop for a supported React Spectrum component differs slightly from the typical event handler prop pattern that you may be familiar with.
-Instead of providing each event handler directly to the component, you must first import `useDragHooks` and `useDropHooks` from the `@react-spectrum/dnd` package
-and provide those hooks with your desired options. `useDragHooks` will then provide you with a set of drag hooks that you can pass to the component via its `dragHooks` prop, thus enabling drag operations for the component.
-Drop operations are then enabled in the same way by passing the hooks from `useDropHooks` to the component's `dropHooks` prop. This approach allows the drag and drop implementation to be included only when used,
+Instead of providing each event handler directly to the component, you must first import `useDnDHooks` from the `@react-spectrum/dnd` package and provide this hook with your desired options.
+`useDnDHooks` will then provide you with a set of drag hooks that you can pass to the component via its `dragHooks` prop, thus enabling drag operations for the component.
+Drop operations are then enabled in the same way by passing the drop hooks from `useDnDHooks` to the component's `dropHooks` prop. This approach allows the drag and drop implementation to be included only when used,
 keeping bundle sizes small when unused by an application.
 
-### useDragHooks
+### useDnDHooks
 
-<FunctionAPI function={docs.exports.useDragHooks} links={docs.links} />
-<TypeContext.Provider value={docs.links}>
-  <InterfaceType properties={docs.exports.DragHookOptions.properties} />
-</TypeContext.Provider>
+TODO: Update this section to better separate the util handlers from the lower level props
```

Comment:
Refinement to the docs for the new hook to be done in a separate PR

Reply:
Is this still relevant?

Reply:
yep, the changes here are minimal, larger docs changes to come in a separate PR